src/sample/index.ts

```diff
@@ -62,15 +69,15 @@ export class ExampleStack extends cdk.Stack {
     });
 
     const datadogCDK = new Datadog(this, "Datadog", {
-      nodeLayerVersion: 39,
-      pythonLayerVersion: 24,
+      nodeLayerVersion: 62,
+      extensionLayerVersion: 10,
       forwarderArn: "<forwarder_ARN>",
       enableDatadogTracing: true,
       flushMetricsToLogs: true,
       apiKey: "1234",
       site: "datadoghq.com",
     });
-    datadogCDK.addLambdaFunctions([hello, hello1, hello2]);
+    datadogCDK.addLambdaFunctions([hello, helloNodeJS, hello1, hello2]);
```

Comment:
I think these function names are potentially confusing. They were already this way before, but now that we're adding another one it's a good opportunity to clarify this. Just looking at the names, it's not obvious what makes each function different. I think we should re-evaluate what all of these functions are called so that it's more clear what each of them is for.

Another thing worth thinking about is that including "NodeJS" by itself in the function name might not communicate that this function's significance is that it is created with the NodeJSFunction construct. Someone might just think it's called "helloNodeJS" because it's using the Node runtime (as opposed to Python or something else)

Reply:
Great idea! Although both functions would use a lot of the same code, I added some helper functions to make this distinction/available method clear.